redash/models.py

```diff
@@ -806,7 +806,18 @@ def create(cls, **kwargs):
 
     @classmethod
     def all_queries(cls, group_ids, user_id=None, drafts=False):
-        q = (cls.query.join(User, Query.user_id == User.id)
+        select_columns = [
+            Query.id,
+            Query.name,
+            Query.created_at,
+            Query.schedule,
+            Query.is_draft,
+            User.name.label("created_by"),
+            QueryResult.retrieved_at,
+            QueryResult.runtime
+            ]
+        q = (db.Query(select_columns, session=SignallingSession(db))
+            .join(User, Query.user_id == User.id)
```

Comment:
SQLAlchemy has a way to eagerly load associated models, see the implementation in my [pull request](https://github.com/getredash/redash/pull/1703/files#diff-2eb9bad8b59774e58b2c9a664440905aR811).